hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/CreateDagVisitor.java

```diff
@@ -186,6 +187,32 @@ public Vertex onMapIndexScan(IndexScanMapPhysicalRel rel) {
                 );
     }
 
+    public Vertex onCalc(CalcPhysicalRel rel) {
+        // TODO [sasha] : program might be trivial [program.isTrivial()], we may just skip vertex creation
```

Comment:
Instead of doing this, we use `CoreRules.CALC_REMOVE` that does exactly this. I think we can remove this TODO.